modules/pages-and-forms/pages/pro-declare-interface-in-bonita.adoc

```diff
@@ -0,0 +1,27 @@
+= Declare the interface in Bonita
+:page-aliases: ROOT:pro-declare-interface-in-bonita.adoc
+:description: This page defines the steps to declare your interface and display it in Bonita.
+
+{description}
+
+== Declare the interface in Bonita application descriptor
+
+The xref:applications:application-creation.adoc[Bonita application descriptor] is a file that let you describe all your applications.
```

Comment:
**nitpick:** 
The link of `xref:applications:application-creation.adoc[Bonita application descriptor]` is not working.